### PR TITLE
Add JF_GIT_USERNAME to bitbucket server

### PR DESCRIPTION
**Pull request by omerzi**
- [x] All [tests](https://github.com/jfrog/frogbot#tests) passed. If this feature is not already covered by the tests, I added new tests.
- [x] This pull request is on the dev branch.
- [x] I used gofmt for formatting the code before submitting the pull request.
---

### CODE REVIEW

File: commands/utils/params.go [hunk 1]
```diff
@@ -209,7 +217,7 @@ func extractGitParamsFromEnv() (Git, error) {
 	if err = readParamFromEnv(GitTokenEnv, &gitParams.Token); err != nil {
 		return Git{}, err
 	}
-
+	_ = readParamFromEnv(GitUsernameEnv, &gitParams.Username)
```
Comment (eyalbe4) [inline commands/utils/params.go:220]: Let's add a comment, explaining why we're ignoring the error returned,
The same goes for the other calls to this function where we ignore the error.

File: go.mod [hunk 2]
```diff
@@ -97,8 +97,8 @@ require (
 	gopkg.in/yaml.v2 v2.4.0 // indirect
 )
 
-replace github.com/jfrog/jfrog-client-go => github.com/jfrog/jfrog-client-go v1.24.6-0.20230102092554-c56fc329c8fc
-
-replace github.com/jfrog/jfrog-cli-core/v2 => github.com/jfrog/jfrog-cli-core/v2 v2.27.1-0.20230103162751-74a639242aeb
-
-replace github.com/jfrog/build-info-go => github.com/jfrog/build-info-go v1.8.5-0.20230102085833-456b7aea6098
+//replace github.com/jfrog/jfrog-client-go => github.com/jfrog/jfrog-client-go v1.24.6-0.20230102092554-c56fc329c8fc
+//
```
Comment (eyalbe4) [inline go.mod:101]: Let's remove these // here.

File: go.mod [hunk 3]
```diff
@@ -97,8 +97,8 @@ require (
...
+//replace github.com/jfrog/jfrog-cli-core/v2 => github.com/jfrog/jfrog-cli-core/v2 v2.27.1-0.20230103162751-74a639242aeb
+//
```
Comment (eyalbe4) [inline go.mod:103]: Let's remove these // here.

**Review by eyalbe4 — CHANGES_REQUESTED** [review]

File: go.mod (hunk 3 above)
Comment (omerzi) [reply go.mod]: done

File: commands/utils/params.go (hunk 1 above)
Comment (omerzi) [reply commands/utils/params.go:221]: done

File: go.mod (hunk 2 above)
Comment (omerzi) [reply go.mod]: done

**Pull request merged**